sympy/matrices/tests/test_matrices.py

```diff
@@ -17,6 +17,9 @@
 from sympy.utilities.pytest import raises, XFAIL, slow, skip
 from sympy.solvers import solve
 from sympy.assumptions import Q
+import warnings
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+warnings.filterwarnings("ignore", category=SymPyDeprecationWarning)
```

Comment:
So may be in test_matrices.py line 323 we should reverted ```assert M.det(method="bareis") == -1``` back to ```assert M.det(method="bareiss") == -1``` after removing this.